src/Components/v2/Lightbox/Lightbox.tsx

```diff
@@ -70,7 +74,7 @@ export interface LightboxState {
 }
 
 @track({ context_module: Schema.ContextModule.Zoom })
-export class Lightbox extends React.Component<LightboxProps, LightboxState> {
+class LightboxComponent extends React.Component<LightboxProps, LightboxState> {
```

Comment:
Why change the name here? We should avoid encoding implementation details if possible.

Reply:
Oh I see, it's for below 👍

Reply:
Is there a different naming convention you'd prefer?

Reply:
No, not necessarily! Another way to do it could be 

```tsx
export const Lightbox = withSystemContext(class extends React.Component<...> {
  ...
})
```
But all in all this is totally fine as is.